Xamarin.Forms.Platform.iOS/Renderers/SliderRenderer.cs

```diff
@@ -152,13 +162,51 @@ protected override void OnElementPropertyChanged(object sender, PropertyChangedE
 				UpdateThumbImage();
 			else if (e.PropertyName == Slider.ThumbColorProperty.PropertyName)
 				UpdateThumbColor();
+			else if (e.PropertyName == Specifics.UpdateOnTapProperty.PropertyName)
+				UpdateTapRecognizer();
 		}
 
 		void OnControlValueChanged(object sender, EventArgs eventArgs)
 		{
 			((IElementController)Element).SetValueFromRenderer(Slider.ValueProperty, Control.Value);
 		}
 
+		void UpdateTapRecognizer()
+		{
+			if (Element != null && Element.IsSet(Specifics.UpdateOnTapProperty))
```

Comment:
Iwould try avoid nesting  `if (Element == null || Control == null || !Element.IsSet(Specifics.UpdateOnTapProperty)) return; `

and save the check on the bottom on 185

Reply:
The check on 185 is for the gesture recognizer callback though. It was needed since tap event was received when the control already has been removed in certain cases.